examples/miscellaneous/plot_outlier_detection_bench.py

```diff
@@ -314,6 +316,11 @@ def fit_predict(estimator, X):
 # datasets. The score for IForest is slightly better for the SA dataset and LOF
 # performs considerably better on the Ames housing dataset than IForest.
 #
+# Recall however that Isolation Forest tends to train much faster than LOF on
+# datasets with a large number of samples. Indeed LOF needs to compute pairwise
+# distances to find nearest neighbors, which has a quadratic complexity in large
+# dimensions. This can make this method prohibitive on large datasets.
```

Comment:
I realize that my earlier suggested phrasing was confusing:

```suggestion
# distances to find nearest neighbors, which has a quadratic complexity with respect
# to the number of observations when the number of features is large and prevents
# the use of tree-based neighbor indexing. This can make this method prohibitive
# on large datasets.
```

Reply:
or simply:

```suggestion
# distances to find nearest neighbors, which has a quadratic complexity with respect
# to the number of observations. This can make this method prohibitive on large
# datasets.
```